docs: update Ubuntu support version (#763)
beclab/Olares: README.md

| Platform            | Operating system                     | Notes                                                 |
|---------------------|--------------------------------------|-------------------------------------------------------|

> | Linux               | Ubuntu 20.0.4 or later <br/> Debian 12.8 or later |                                          |

Reviewer: ```suggestion
| Linux               | Ubuntu 20.04 LTS or later <br/> Debian 11 or later |  Verified on Ubuntu 20.04 and Debian 12.8                                        |
```

| Raspberry Pi        | RaspbianOS                           | Verified on Raspberry Pi 4 Model B and Raspberry Pi 5 |

> | Windows             | Windows 11 23H2 or later <br/>Windows 10 22H2 or later |                                     |

Reviewer: ```suggestion
| Windows             | Windows 11 23H2 or later <br/>Windows 10 22H2 or later |                                     |
```
Shoud we specify the exact version? Or should we mention WSL?

> | Mac (Apple silicon) | Monterey (12) or later              |                                                        |

Reviewer: ```suggestion
| Mac | Monterey (12) or later              |                                                        |
```
Should we specify apple silicon or Intel?
